charts/invenio/templates/worker-beat-deployment.yaml

```diff
@@ -29,7 +29,14 @@ spec:
   template:
     metadata:
       labels:
+        {{- with .Values.workerBeat.podLabels }}
+          {{- tpl (toYaml .) | nindent 10 }}
+        {{- end }}
         app.kubernetes.io/component: worker-beat
+      {{- with .Values.workerBeat.podAnnotations }}
+      annotations:
+        {{- tpl (toYaml .) | nindent 8 }}
+      {{- end }}
```

Comment:
Why is `nindent` different for labels and annotations? 🤔 

Also, as a very minor nitpick: Can we have annotations first just so the fields are sorted alphabetically?